src/lib/containers/EntityBadgeIcons.tsx

```diff
@@ -1,15 +1,14 @@
 import React, { useState } from 'react'
 import { EntityBundle, EntityType } from '../utils/synapseTypes'
 import {
-  faAlignLeft,
-  faCheck,
-  faComment,
-  faGlobeAmericas,
-  faLock,
-  faTag,
-  faUnlink,
-} from '@fortawesome/free-solid-svg-icons'
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+  DescriptionTwoTone,
```

Comment:
Initially I was pulling material icons directly, but realized that the IconSvg wrapper was useful in a number of contexts (to add styling to a wrapper).  So there's still a mix